.github/workflows/build-release.yml

```diff
@@ -209,4 +209,4 @@ jobs:
           artifactErrorsFailBuild: true
           tag: "${{ inputs.releaseVersion }}"
           name: "v${{ inputs.releaseVersion }}"
-          artifacts: "*.csv,*.json,*.xlsx"
+          artifacts: "*.csv,*.json,*.xlsx,*.yaml"
```

Comment:
Anyone know of a good way to test this before actually doing a release? Seems straightforward, but it would be nice to validate.

Reply:
@aschey-forpeople I went on your branch and ran mvn clean install -DskipITs -DskipTests=true -Dmaven.build.cache.enabled=false -Dapidocgen.skip=false then under the dist directory you can checkout OpenAPI-2.118.0-SNAPSHOT.yaml. I can see your changes there

Reply:
@meliGuzman thanks for checking! I was specifically referring to testing that the yaml doc got attached to the github release. I suppose I can run a one-off github action that runs a build and creates a dummy release that we can delete later, but that sounds annoying.

Reply:
yeahh that might be the way to go as far as I know